### PR TITLE
[Bug] Fixed logo not redirecting to '/' (homepage)

### PR DESCRIPTION
**Pull request by ahmedsilat44**
## Related Issue

- Information about the related issue

Closes: #194 

### Describe the changes you've made

I added <a></a> to redirect to '/' (the homepage) when clicking on the WebXDAO Logo.

Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/201"></a>

### CODE REVIEW

**Comment by ahmedsilat44** [comment]
Hey @Panquesito7 could you help me fix the tests that failed 😟 . Thanks.

File: src/components/Global/Navbar.js [hunk 1]
```diff
@@ -24,14 +24,14 @@ const Navbar = () => {
           <div className="flex flex-1 items-center">
             <div className="flex w-full items-center justify-between md:w-auto">
               <Link href={'/'}>
-                <>
-                  <span className="sr-only">WebX DAO</span>
+                <a href="/">
+                  <span className="sr-only ">WebX DAO</span>
```
Comment (Panquesito7) [inline src/components/Global/Navbar.js:28]: This might be causing the issue.
```suggestion
                  <span className="sr-only">WebX DAO</span>
```

**Review by krshkodes — CHANGES_REQUESTED** [review]
@ahmedsilat44 The error is caused, because you do not use the a tag in NextJS but the Link tag, everything else is fine, just import the Link Component from next/link and replace that with the a tag

**Comment by ahmedsilat44** [comment]
@krshkodes I ran npm run build after changing and it worked. Could you review it now. Thanks 😄

**Comment by mkubdev** [comment]
> @krshkodes I ran the npm run build after changing and it worked. Could you review it now? Thanks 😄

Hi, you can't ask maintainers to review your changes 'now'. We all work accross the world and have different timezone so if you wait a moment, your PR will be obviously checked. ☺️

**Review by mkubdev — COMMENTED** [review]
Look good to me. 🚀

**Comment by ahmedsilat44** [comment]
> > @krshkodes I ran the npm run build after changing and it worked. Could you review it now? Thanks 😄
> 
> 
> 
> Hi, you can't ask maintainers to review your changes 'now'. We all work accross the world and have different timezone so if you wait a moment, your PR will be obviously checked. ☺️

I'm sorry. I didn't mean it like that 😅

**Comment by mkubdev** [comment]
> > > @krshkodes I ran the npm run build after changing and it worked. Could you review it now? Thanks 😄
> > 
> > 
> > Hi, you can't ask maintainers to review your changes 'now'. We all work accross the world and have different timezone so if you wait a moment, your PR will be obviously checked. ☺️
> 
> I'm sorry. I didn't mean it like that 😅

No problem! :p

File: src/components/Global/Navbar.js [hunk 2]
```diff
@@ -24,14 +24,14 @@ const Navbar = () => {
           <div className="flex flex-1 items-center">
             <div className="flex w-full items-center justify-between md:w-auto">
               <Link href={'/'}>
-                <>
+                <a>
```
Comment (krshkodes) [inline src/components/Global/Navbar.js:27]:
```suggestion
                <Link href="/">
```

File: src/components/Global/Navbar.js [hunk 3]
```diff
@@ -24,14 +24,14 @@ const Navbar = () => {
...
                   <span className="sr-only">WebX DAO</span>
                   <img
-                    className="h-8 w-auto sm:h-10"
+                    className="h-8 w-auto sm:h-10 cursor-pointer"
                     src="/images/logo/white_logo.png"
                     alt="WebX DAO White Logo"
                   />
-                </>
+                </a>
```
Comment (krshkodes) [inline src/components/Global/Navbar.js:34]:
```suggestion
                </Link>
```

**Comment by ahmedsilat44** [comment]
> Also add this import at the top of the file:
> 
> `import Link from 'next/link'`

This is already present at the top of the file.

**Review by krshkodes — CHANGES_REQUESTED** [review]
Hi you need to import the Link component from next/link as I told in my previous review

**Comment by ahmedsilat44** [comment]
> Hi you need to import the Link component from next/link as I told in my previous review

Hi @krshkodes sorry I am really confused. The line of code you are telling me to add already exists in the file, on the 2nd line. Sorry for the repeated back and forth 😓 

image

File: src/components/Global/Navbar.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import { Fragment } from "react";
 import Link from 'next/link';
+import { Fragment } from "react";
 import { Popover, Transition } from "@headlessui/react";
 import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
 import { SiGithubsponsors } from "react-icons/si";
```
Comment (krshkodes) [inline src/components/Global/Navbar.js:5]:
```suggestion
import { Fragment } from "react";
import { Popover, Transition } from "@headlessui/react";
import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
import { SiGithubsponsors } from "react-icons/si";
import Link from 'next/link';
```

Let's try to keep the git log to the minimum

**Review by krshkodes — CHANGES_REQUESTED** [review]

**Comment by krshkodes** [comment]
Oh Sorry @ahmedsilat44 I think I missed that line and didn't see it, my bad

**Review by krshkodes — APPROVED** [review]

**Pull request merged**